pkg/balancer-js/src/pool-stable/encoder.ts

```diff
@@ -5,11 +5,13 @@ export enum StablePoolJoinKind {
   INIT = 0,
   EXACT_TOKENS_IN_FOR_BPT_OUT,
   TOKEN_IN_FOR_EXACT_BPT_OUT,
+  ALL_TOKENS_IN_FOR_EXACT_BPT_OUT,
 }
 
 export enum StablePoolExitKind {
   EXACT_BPT_IN_FOR_ONE_TOKEN_OUT = 0,
   BPT_IN_FOR_EXACT_TOKENS_OUT,
+  EXACT_BPT_IN_FOR_TOKENS_OUT,
```

Comment:
```suggestion
  EXACT_BPT_IN_FOR_TOKENS_OUT,
  BPT_IN_FOR_EXACT_TOKENS_OUT,
```

We should use the same ids we used in the past

Reply:
(This change needs to be done in multiple places)